plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/tree/expressions.kt

```diff
@@ -72,6 +72,15 @@ class JKQualifiedExpression(
     override fun accept(visitor: JKVisitor) = visitor.visitQualifiedExpression(this)
 }
 
+class JKArrayGetterExpression(
+    receiver: JKExpression,
+    literal: JKExpression,
+    override val expressionType: JKType? = null,
+) : JKQualifiedExpression(receiver, literal) {
```

Comment:
Both `KtArrayAccessExpression` and `PsiArrayAccessExpression` are not qualified expressions.